projects/libavc/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -eu
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Build libavc
+build_dir=$WORK/build
+rm -rf ${build_dir}
+mkdir -p ${build_dir}
+pushd ${build_dir}
+
+cmake $SRC/libavc
```

Comment:
I believe this should have:

```
    -DCMAKE_C_COMPILER="${CC}" \
    -DCMAKE_CXX_COMPILER="${CXX}" \
    -DCMAKE_C_FLAGS="${CFLAGS}" \
    -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
````
... to ensure that the library is properly build with coverage + applicable sanitizers. @Dor1s can you confirm?

Reply:
Doesn't seem necessary.

Verified in local setup of oss-fuzz by adding VERBOSE=1 to make command and confirmed that sanitizer flags are appropriately passed while compiling the sources in library.

https://cmake.org/cmake/help/v3.12/envvar/CFLAGS.html also says CFLAGS will be used

However, will add these, if this is the recommended way.

Reply:
Yes unneeded.

Reply:
> @Dor1s can you confirm?

It depends on a particular cmake file, e.g. some might be ignoring env vars and expect them to be provided explicitly. @harishdm thanks for verifying.

Usually the bad build check would not pass and block the PR if the flags weren't propagated properly.